### ISSUE TITLE
consider unique content localized content on advocacy homepage

### ISSUE THREAD

**Issue by ScottDowne** [MozillaFoundation/Advocacy#51]
The TLDR we should change up the content on advocacy.mozilla.org's homepage to be based on the locale, and have a way to update this easily.

This is more of a brainstorm idea, but I think it might have potential.

Right now on advocacy.mozilla.org's homepage, we showcase various campaigns and projects.

Examples, the encrypt campaign, Open Web Fellows, Tools for Activism, and USA FREEDOM Act.

What's interesting about localizing the site, is we can now look at these showcased campaigns and projects under a different context.

Example, it probably doesn't make much sense to localize the content in the USA FREEDOM Act and show that to people outside of the USA.

So the content on this page shouldn't be static and should be managed somehow, kinda like how the FF homepage ships different snippets per different locales.

Thoughts? @brettgaylor @valianttry

**Comment by valianttry** [comment]
I think the idea is good, but this seems like a very low priority. We don't send much traffic to this homepage at all. We send people to campaigns / specific tools -- which should be prioritized over the homepage based on traffic.

**Comment by ScottDowne** [comment]
I'm fine with closing this.
